.github/workflows/footprint-tracking.yml

```diff
@@ -35,6 +26,9 @@ jobs:
       ZEPHYR_SDK_INSTALL_DIR: /opt/toolchains/zephyr-sdk-0.13.2
       CLANG_ROOT_DIR: /usr/lib/llvm-12
       ZEPHYR_TOOLCHAIN_VARIANT: zephyr
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.run_id }}
```

Comment:
I think such usage is obsolete, since github.run_id is unique for each, run hence such condition will never appear